src/common/CustomTable/CustomTable.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+import { EuiBasicTable } from '@elastic/eui';
+
+const CustomTable = ({
+  items, columns, sorting, onChange,
```

Comment:
If you define `rest` then you won't need `items` and `columns` here. Same as for `CustomIconButtom`

Reply:
Fixed